tests/unit/synapseclient/unit_test_client.py

```diff
@@ -2543,3 +2544,21 @@ def test_query_table_csv(self, downloadLocation, syn):
             )
 
             assert (mock_download_result, expected_path) == actual_result
+
+
+def test_init_change_cache_path():
+    cache_root_dir = '.synapseCache'
```

Comment:
It's good to have a docstring as the first line of a test that describes what it is doing.